ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/sequence/TrajectoryDurationTracker.java

```diff
@@ -40,13 +47,28 @@ public boolean getTimeIsUp()
       return timeIsUp;
    }
 
-   public boolean getHitTimeLimit()
+   public boolean getHitTimeLimit(BehaviorTreeNodeMessageLogger logger)
    {
+      if (hitTimeLimit)
+      {
+         logger.error(1, "Task execution timed out. %.2f > %.2f Nominal: %.2f".formatted(executionTimer.getElapsedTime(), timeout, nominalExecutionDuration));
```

Comment:
I put this error message in this call so the operator gets more info about the failure, without having to adjust the print for the 7 usages.